edge-modules/TestAnalyzer/Reporter.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Extensions.Logging;
+
+    class Reporter
+    {
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("Analyzer");
```

Comment:
```suggestion
        static readonly ILogger Logger = ModuleUtil.CreateLogger(nameof(Reporter));
```

Reply:
I'll correct all instances of this pattern.